testing/src/main/java/io/stargate/it/http/MetricsTest.java

```diff
@@ -66,7 +67,13 @@ public void restApiHttpRequestMetrics() throws IOException {
             .get()
             .addHeader(TagMeHttpMetricsTagProvider.TAG_ME_HEADER, "test-value")
             .build();
-    Response execute = client.newCall(request).execute();
+
+    MutableInt status = new MutableInt();
```

Comment:
Maybe just `AtomicInteger`?

Reply:
`MutableInt` is a good fit here. `AtomicInteger` says this might be used on another thread and would cause me to go looking around for multi-threaded stuff.

Reply:
Yes, sure. But `MutableInt` is a class from an external (3rd party) library, whereas the `AtomicInteger` already exists in the JDK.

Reply:
`int` will work, no idea how `MutableInt` came on my keyboard :exploding_head: